core/templates/dev/head/filters/string-utility-filters/camel-case-to-hyphens.pipe.spec.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for CamelCaseToHyphens pipe for Oppia.
+ */
+
+import { CamelCaseToHyphensPipe } from
+  'filters/string-utility-filters/camel-case-to-hyphens.pipe';
+
+describe('Testing pipes', () => {
```

Comment:
Add the pipe name in the describe string as this is used in the error log!

Reply:
Isn't it there `Testing pipes`?

Reply:
So the error log will state `Failed Testing pipes should have all expected pipes`, which doesn't state which pipe is failing the test.  I think the describe message should include the pipe name.

In general, the `describe` message + `it` message should be a complete sentence explaining the specific test.

Reply:
Yeah, right. Thanks for the explanation.

Reply:
Done